src/components/KanbanCard.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="card">
+    <div class="card">
+      <div id="Back-Log" class="card-header" v-if="data.name==`Back-Log`">{{data.name}}</div>
```

Comment:
Indentasinya di rapihkan 

```suggestion
      <div id="Back-Log" class="card-header" v-if="data.name === `Back-Log`">{{data.name}}</div>
```